A/ApproxFunBase/Compat.toml

```diff
@@ -6,19 +6,27 @@ BlockBandedMatrices = "0.4"
 Calculus = "0.1.15-0.4"
 DSP = "0.5.1-0.5.2"
 DomainSets = "0.0"
-DualNumbers = "0.4-0.6"
 FFTW = "0.2.4"
 FastGaussQuadrature = "0.3.2-0.3.3"
 FastTransforms = "0.4.2"
-FillArrays = "0.5"
 InfiniteArrays = "0.0.3"
 IntervalSets = "0.3.1"
-LazyArrays = "0.7.1-0.8"
 LowRankApprox = "0.1.4-0.2"
 SpecialFunctions = "0.7"
 StaticArrays = "0.8.3-0.10"
 ToeplitzMatrices = "0.4-0.5"
-julia = "0.7-1.3"
 
 ["0.0.1"]
 RecipesBase = "0.5-0.6"
+
+["0.0.1-0.0.3"]
+DualNumbers = "0.4-0.6"
+FillArrays = "0.5"
+LazyArrays = "0.7.1-0.8"
+julia = "0.7-1.3"
+
+["0.0.4"]
+DualNumbers = "0.6.2"
+FillArrays = "0.5-0.6"
+LazyArrays = "0.8"
+julia = "0.7-1.1"
```

Comment:
This is wrong, see https://github.com/JuliaComputing/Registrator.jl/issues/103. Linking it with that issue so we can correct it later.